3587/top_gems.rb

```diff
@@ -0,0 +1,53 @@
+require 'mechanize'
+require 'optparse'
+require './links_modifier.rb'
+require './scrape_info.rb'
+require './fill_output.rb'
+# class that contain main info about gem
+class GemStart
+  def initialize
+    @options = {}
+    @array_of_gems = []
+    @parser = OptionParser.new do |opt|
```

Comment:
Why I can see this in initializer? You can send Option parser object as an argument for GemStart initializer